lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -62,11 +66,30 @@ def line2d_seg_dist(p1, p2, p0):
     return d
 
 
-def mod(v):
+@cbook.deprecated("3.1")
+def line2d_seg_dist(p1, p2, p0):
+    """distance(s) from line defined by p1 - p2 to point(s) p0
+
+    p0[0] = x(s)
+    p0[1] = y(s)
+
+    intersection point p = p1 + u*(p2-p1)
+    and intersection point lies within segment if u is between 0 and 1
+    """
+    return _line2d_seg_dist(p1, p2, p0)
+
+
+def _mod(v):
     """3d vector length"""
     return np.sqrt(v[0]**2+v[1]**2+v[2]**2)
 
 
+@cbook.deprecated("3.1")
+def mod(v):
+    """3d vector length"""
+    return _mod(v)
```

Comment:
All of these `mod` related things should be superceded by #13020, because it is straight-forward to just use `np.linalg.norm()`.